mktxp/collector/switch_collector.py

```diff
@@ -24,16 +24,17 @@ def collect(router_entry):
             return
 
         switch_port_labels = ['name', 'driver_rx_byte', 'driver_rx_packet', 'driver_tx_byte',  'driver_tx_packet', 
-                              'rx_bytes', 'rx_broadcast', 'rx_pause', 'rx_multicast', 'rx_fcs_error', 'rx_align_error', 'rx_fragment', 'rx_overflow', 
-                              'tx_bytes', 'tx_broadcast', 'tx_pause', 'tx_multicast', 'tx_underrun', 'tx_collision', 'tx_deferred']
+                              'rx_bytes', 'rx_unicast', 'rx_broadcast', 'rx_pause', 'rx_multicast', 'rx_fcs_error', 'rx_align_error', 'rx_fragment', 'rx_overflow', 
+                              'tx_bytes', 'tx_unicast', 'tx_broadcast', 'tx_pause', 'tx_multicast', 'tx_underrun', 'tx_collision', 'tx_deferred', 'tx_drop']
 
         switch_port_records = SwitchPortMetricsDataSource.metric_records(router_entry, metric_labels = switch_port_labels)   
         if switch_port_records:
             yield BaseCollector.counter_collector('switch_driver_rx_byte', 'Total count of received bytes', switch_port_records, 'driver_rx_byte', ['name'])
             yield BaseCollector.counter_collector('switch_driver_rx_packet', 'Total count of received packets', switch_port_records, 'driver_rx_packet', ['name'])
             yield BaseCollector.counter_collector('switch_driver_tx_byte', 'Total count of transmitted bytes', switch_port_records, 'driver_tx_byte', ['name'])
-            yield BaseCollector.counter_collector('switch_driver_tx_packet', 'Total count of transmitted packets', switch_port_records, 'driver_tx_packet', ['name'])
+            yield BaseCollector.counter_collector('switch_driver_tx_packet', 'Total count of transmitted packets', switch_port_records, 'driver_tx_packet', ['name'])          
```

Comment:
@smitas3400 thanks for the PR. A quick question, are the new metrics present on all / most mikrotik devices? 
since if not, it'd be necessary to check on them before yielding (see e.g. [this](https://github.com/akpw/mktxp/blob/main/mktxp/collector/health_collector.py) for an example